docs/install.md

```diff
@@ -43,6 +43,8 @@ Compose can also be installed as a Python package:
 No further steps are required; Compose should now be successfully installed.
 You can test the installation by running `docker-compose --version`.
 
+> Note: If you get an error like "Illegal instruction: 4" after installing using the binary on Mac OS X, your CPU architecture isn't supported and you should install using Pip instead.
```

Comment:
@aanand move this note after the sentence starting "Optionally,..."

Change Note: to

 **Note**: Some older Mac OS X CPU architectures are incompatible with  installing the binary using `curl`.  If you receive an "Illegal instruction: 4" error after installing using `curl`, you should install using the `pip` command instead.